docs/how-to/h-deploy-spark-history-server.md

```diff
@@ -0,0 +1,96 @@
+## Deploy the Spark History Server 
+
+The Spark History server is a component in the Spark ecosystem that helps you monitor your spark jobs. In particualar, the History Server enables access to logs of completed/running Spark jobs for viewing and analysis.
+
+Within the Charmed Spark solution, its deployment, configuration and operation is handled by [Juju](https://juju.is/). Therefore, a [Charmed operator](https://charmhub.io/spark-history-server-k8s) will be responsible for its management and operations on K8s.
```

Comment:
Maybe say "Therefore, a Juju charm(link to charm) will be responsible for its management and operations on K8s." Then provide a short explanation of what a Juju charm is. I think there is a standard explanation text that Massi and Jon recently prepared.

Reply:
Maybe also worth mentioning that there's a Juju provider for Terraform that users can leverage to integrate Juju with their existing systems management and provide a link. We should probably do that in all of our documentation.

Reply:
Uhm, not sure about this. I mean, this is a how-to guide where we should not provide either explanations or other extra information than a step by step guide to get their task done. Actually I definitely agree that it would be good to provide a terraform template, but probably in a separate topic that this one